modules/resources/src/main/res/values-pt/strings.xml

```diff
@@ -9,7 +9,7 @@
     <string name="main_dialog_text">Digite o número do WhatsApp com o qual deseja iniciar uma conversa.</string>
     <string name="main_dialog_negative_button">Cancelar</string>
     <string name="main_dialog_positive_button">Iniciar conversa</string>
-    <string name="main_dialog_input_label">Número do telefone</string>
+    <string name="main_dialog_input_label">Digite o número</string>
```

Comment:
Not sure if this is the ideal text. The supporting text is already saying "Digite o número ...".
Maybe I should address this issue on the code side by adding an ellipsis in case the text does not fit.

Do you have another suggestion for this text? Otherwise, I'm inclined to decline the PR and solve the problem on the code side.

Reply:
Or you could add a dynamic size text.

Reply:
https://developer.android.com/develop/ui/views/text-and-emoji/autosizing-textview

Reply:
Jetpack Compose does not have auto-sizing text yet, so I would have to build (or copy) an implementation of it... I don't think it's worth it. Besides, the label isn't just a simple text that I can apply such property, it's part of a bigger component, AKA TextField that respects m3 guidelines. 

@marciozomb13  Could you please open an issue with an image of the line breaking? I'll work on it ASAP.
I hate having this kind of bug in the app.